script/tasks/diamondUpdatePeriphery.sh

```
@@ -209,9 +209,9 @@ register() {
     else
       # do not print output to console
       if [[ "$ENVIRONMENT" == "production" ]]; then
-        # set DEPLOY_NEW_NETWORK_MODE to true when deploying a new network so that transactions are not proposed to SAFE (since deployer is still the diamond contract owner during deployment)
-        if [ "$DEPLOY_NEW_NETWORK_MODE" == "true" ]; then
-          echo "DEPLOY_NEW_NETWORK_MODE is activated - registering "$CONTRACT_NAME" as periphery on diamond "$DIAMOND_ADDRESS" in network $NETWORK now..."
+        # set SEND_PROPOSALS_DIRECTLY_TO_DIAMOND to true when deploying a new network so that transactions are not proposed to SAFE (since deployer is still the diamond contract owner during deployment)
+        if [ "$SEND_PROPOSALS_DIRECTLY_TO_DIAMOND" == "true" ]; then
+          echo "SEND_PROPOSALS_DIRECTLY_TO_DIAMOND is activated - registering "$CONTRACT_NAME" as periphery on diamond "$DIAMOND_ADDRESS" in network $NETWORK now..."
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)